### PR TITLE
re add completed courses parsing

### PR DESCRIPTION
**Pull request by NilsBacke**

### CODE REVIEW

**Comment by NilsBacke** [comment]
[Reconnect Completed Courses parsing](https://trello.com/c/n7LlBnQk/317-reconnect-completed-courses-parsing)

**Review by imanmoreira — CHANGES_REQUESTED** [review]
Rn it's setting the completed course calendar correctly but it's not actually using it for their initial plan (and just randomly parsing in the completed courses for their initial plan). 
We wanna make sure it's hooked up correctly for after they log in for that first time

File: frontend/src/Onboarding/TransferableCreditScreen.tsx [hunk 1]
```diff
@@ -198,22 +198,22 @@ const TransferableCreditScreen: React.FC = () => {
...
-          allPlans
-        );
-      } else {
-        schedule = generateBlankCompletedCourseScheduleNoCoopCycle(
-          academicYear,
-          graduationYear,
-          completedCourseSchedule!
-        );
-      }
+      // if (!!coopCycle) {
```
Comment (andrewjleung) [inline frontend/src/Onboarding/TransferableCreditScreen.tsx:201]: Why did we get rid of this condition?

**Comment by andrewjleung** [comment]
Slightly confused on the context for this PR, the ticket description is cut off. What are these changes supposed to be doing?

File: frontend/src/Onboarding/TransferableCreditScreen.tsx (hunk 1 above)
Comment (imanmoreira) [reply frontend/src/Onboarding/TransferableCreditScreen.tsx:201]: This condition was adding an error, so it was just a temporary fix

**Review by imanmoreira — APPROVED** [review]

**Pull request merged**